README.md

```diff
@@ -1,20 +1,66 @@
+![GitHub](https://img.shields.io/github/license/HumanCellAtlas/secondary-analysis.svg?style=flat-square&colorB=blue)
+[![Code style: black](https://img.shields.io/badge/Code%20Style-black-000000.svg?style=flat-square)](https://github.com/ambv/black)
+
+This repo is the gateway of the Secondary Analysis Service which is part of the Human Cell Atlas Data Coordination Platform, containing the testing suites, automations and utility scirpts of the Secondary Analysis Service. This repo also serves as a issue tracker and hosting all of the tickets of the Secondary Analysis Service.
 
-This repo will contain integration test scripts and other automation related to the Secondary Analysis Service, which is part of the Human Cell Atlas Data Coordination Platform.
 
 Other Secondary Analysis Service repos:
-* [cromwell-tools](https://github.com/broadinstitute/cromwell-tools): Tools(Python API) for working with the Cromwell workflow engine.
-* [falcon](https://github.com/HumanCellAtlas/falcon): Throttles workflows and start queued workflows.
-* [lira](https://github.com/HumanCellAtlas/lira): Listens to storage service notifications and launches workflows.
-* [pipeline-tools](https://github.com/HumanCellAtlas/pipeline-tools): DCP adapter pipelines and associated tools.
-* [sctools](https://github.com/HumanCellAtlas/sctools): Tools for single cell data analysis.
-* [skylab](https://github.com/HumanCellAtlas/skylab): Analysis pipelines.
-* [skylab-analysis](https://github.com/HumanCellAtlas/skylab-analysis): Analysis and benchmarking reports for Skylab pipelines.
 
+- [Cromwell Tools](https://github.com/broadinstitute/cromwell-tools): A collection of Python clients and accessory scripts for interacting with the [Cromwell workflow execution engine](https://github.com/broadinstitute/cromwell) - a scientific workflow engine designed for simplicity and scalability
+
+- [Falcon](https://github.com/HumanCellAtlas/falcon): Queueing system that (after launching) throttles and inititates workflows 
+
+- [Lira](https://github.com/HumanCellAtlas/lira): Listens to storage service notifications and launches workflows
+
+- [Pipeline Tools](https://github.com/HumanCellAtlas/pipeline-tools): Contains Data Coordination Platform adapter pipelines and associated tools
+
+- [scTools](https://github.com/HumanCellAtlas/sctools): Tools for single cell data processing
+
+- [Secondary Analysis Deploy](https://github.com/HumanCellAtlas/secondary-analysis-deploy): Contains the deployment configuration and scripts for the Pipeline Execution Service
+
+- [Skylab Analysis](https://github.com/HumanCellAtlas/skylab-analysis): Analysis and benchmarking reports for standardized HCA pipelines
+
+- [Skylab](https://github.com/HumanCellAtlas/skylab): Standardized HCA data processing pipelines
+
+## Development
+
+### File Structure Layout
+
+_Note: The following layout is generated by `tree -L 2 -d`_
+```
+.
```

Comment:
Can we remove this `.` here?

Reply:
Sure!

Reply:
Actually removing it results in weird layout above certs, going to add slash after the `.`